src/js/shared/getCSPHeadersFromWebRequestResponse.ts

```diff
@@ -9,7 +9,11 @@ export function getCSPHeadersFromWebRequestResponse(
   response: chrome.webRequest.WebResponseHeadersDetails,
   reportHeader = false,
 ): Array<chrome.webRequest.HttpHeader> {
-  return response.responseHeaders.filter(
+  const responseHeaders = response.responseHeaders;
+  if (!responseHeaders) {
+    throw new Error('Request is missing responseHeaders');
```

Comment:
Does CV go into failure mode in this case? It should.

Reply:
Good question... I don't think it would.

This particular change doesn't actually change that, all it really does is satisfy the compiler (since we're throwing) and change which error would get logged from "trying to call filter on undefined" to "Request is missing responseHeaders".

How do you want to handle cases where these headers are missing?